typescript2/app-website/app/globals.css

```
@@ -530,9 +527,10 @@
   --border: #d8cfbd;
   --input: #d8cfbd;
   --ring: #6d28d9;
+  color: var(--vsc-text);
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)